app/src/main/res/values/translatable.xml

```diff
@@ -358,6 +358,8 @@ THE SOFTWARE.
 	<string name="playlist_sync_disabled">Disabled</string>
 	<string name="playlist_sync_folder_title">Synchronization folder</string>
 	<string name="playlist_sync_folder_summary">Folder to use for playlist synchronization</string>
+	<string name="playlist_sync_relative_paths_title">Export using relative paths</string>
+	<string name="playlist_sync_relative_paths_summary">Reference file paths relative to the playlist</string>
```

Comment:
I'm tempted to suggest to replace 'sync relative' with 'export relative' in this PR.

But i'm also fine with keeping the current wording.

Reply:
I was considering giving it that name initially, but wasn't sure, since synchronization preferences commonly begin with playlist_sync.
I've changed it to be export relative now.